packages/wonder-blocks-button/components/button-core.js

```diff
@@ -11,6 +11,8 @@ import Color, {
     fade,
 } from "@khanacademy/wonder-blocks-color";
 import {addStyle} from "@khanacademy/wonder-blocks-core";
+import {CircularSpinner} from "@khanacademy/wonder-blocks-progress-spinner";
```

Comment:
It would be cool if we could just the spinner more like an icon whenever we implement the `icon` prop, but the fact that the behaviour is so different makes it difficult. 😞